### PR TITLE
WFK2-354 resteasy-spring quickstart

### PR DESCRIPTION
**Pull request by joshuawilson**
This quickstart was started in jboss-eap-quickstarts and we wanted it under the WFK umbrella, so I copied it and it's history.

After bringing it over I made the updates required by QSTools.

It is ready to be merged after a brief review. The README may need some tweaking to match the WFK quickstart style and format.

### CODE REVIEW

**Comment by joshuawilson** [comment]
@sgilda I could squash this if you wanted because it is only the initial work on it and it was never merged into the main repo.

**Comment by sgilda** [comment]
@joshuawilson : I thought you planned to rename all the quickstarts with a leading `spring`, for example,  spring-resteasy? Or is that only for the quickstarts that originate from the Spring examples?

File: resteasy-spring/README.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+Summary: Basic example demonstrating how a spring application can be packaged for JBoss EAP  
+Target Product: WFK  
+Product Versions: EAP 6.1, EAP 6.2, WFK 2.4  
+Source: <https://github.com/jboss-developer/jboss-wfk-quickstarts/>  
+
+What is it?
+-----------
+
+This is a project demonstrates how to package and deploy a web application which includes resteasy-spring integration
+into Red Hat JBoss Enterprise Application Platform.
```
Comment (sgilda) [inline resteasy-spring/README.md]: This project demonstrates how to package and deploy a web application, which includes resteasy-spring integration, into Red Hat JBoss Enterprise Application Platform.

**Comment by sgilda** [comment]
The tests fail for me when I run: mvn clean install jboss-as:deploy integration-test

This is the result: 

Running org.jboss.as.quickstarts.resteasyspring.test.ResteasySpringTest
Tests run: 2, Failures: 2, Errors: 0, Skipped: 0, Time elapsed: 0.25 sec <<< FAILURE!

Results :

Failed tests:   testLocatingResource(org.jboss.as.quickstarts.resteasyspring.test.ResteasySpringTest): expected:<200> but was:<404>
  testHelloSpringResource(org.jboss.as.quickstarts.resteasyspring.test.ResteasySpringTest): expected:<200> but was:<404>

Tests run: 2, Failures: 2, Errors: 0, Skipped: 0

File: resteasy-spring/README.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
+resteasy-spring: Example Using Resteasy Spring Integration
+==========================================================
+Author: Weinan Li <l.weinan@gmail.com>, Paul Gier <pgier@redhat.com>  
+Level: Beginner  
+Technologies: Resteasy, Spring  
+Summary: Basic example demonstrating how a spring application can be packaged for JBoss EAP  
+Target Product: WFK  
+Product Versions: EAP 6.1, EAP 6.2, WFK 2.4
```
Comment (maschmid) [inline resteasy-spring/README.md]: the resteasy-spring artifact is there since EAP 6.2, so EAP 6.1 should be out...  also, WFK 2.5

**Comment by sgilda** [comment]
@joshuawilson : `mvn clean install jboss-as:deploy integration-test` still doesn't run for me. 

I still see:

Running org.jboss.as.quickstarts.resteasyspring.test.ResteasySpringTest
Tests run: 2, Failures: 2, Errors: 0, Skipped: 0, Time elapsed: 0.25 sec <<< FAILURE!

Results :

Failed tests: testLocatingResource(org.jboss.as.quickstarts.resteasyspring.test.ResteasySpringTest): expected: but was:
testHelloSpringResource(org.jboss.as.quickstarts.resteasyspring.test.ResteasySpringTest): expected: but was:

Tests run: 2, Failures: 2, Errors: 0, Skipped: 0

**Comment by joshuawilson** [comment]
I am unsure if squashing it and renaming has caused some difference or not.  I re-did the work and going to submit another PR with the new branch.  

Closing this PR.

**Pull request closed without merging**